src/ametsuchi/wsv.cc

```diff
@@ -161,8 +161,11 @@ void WSV::asset_create(const iroha::AssetCreate *command) {
 
 void WSV::asset_add(const iroha::AssetAdd *command) {
   // Now only Currency is supported
-  if (command->asset_nested_root()->asset_type() != iroha::AnyAsset::Currency)
+  if (command->asset_nested_root()->asset_type() != iroha::AnyAsset::Currency) {
```

Comment:
We still have error here with nested flatbuffers. 
The checks always say that the asset is not a currency